Fluid.Tests/ParserTests.cs

```diff
@@ -994,21 +994,25 @@ public void ShouldParseLiquidTag()
         }
 
         [Fact]
-        public void ShouldParseLiquidTagWithBlocks()
+        public void LiquidTagShouldBreakOnCompletion()
         {
-            var source = @"
-{% liquid assign cool = true
-   if cool
-     echo 'welcome to the liquid tag' | upcase
-   endif 
-%}
-";
+            var source = """
+                {%- for i in (1..5) %}
```

Comment:
Is `-` after the percentage symbol for a reason? This is the first time I see it

Reply:
In liquid it means that the spaces before the tag should not be rendered. Works on every tags or output tag.